src/main/java/seedu/duke/commands/RatingCommand.java

```diff
@@ -10,9 +10,17 @@ public RatingCommand(String showName) {
         this.showName = showName;
     }
 
-    public void rateShow(String showToBeRated, int rating) {
-        Show show = ShowList.getShow(showToBeRated);
-        show.setRating(rating);
-        ShowList.setShow(showName,show);
+    public void rateShow(String showToBeRated, int rating) throws NullPointerException {
+        if (ShowList.getShowList().containsKey(showToBeRated)) {
+            if (rating < 0 || rating > 10) {
+                throw new IndexOutOfBoundsException();
```

Comment:
is it possible to throw more than 1 exception? Does it even catch the IndexOutofBounds?